src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -971,6 +975,162 @@ public static Text of(Object... objects) {
     }
 
     /**
+     * Returns the Text value for the specified key in the specified
+     * {@link Dictionary} for the specified {@link Locale}.
+     *
+     * @param dictionary Dictionary to search
+     * @param key Key to search for
+     * @param locale Locale target
+     * @return Localized text
+     */
+    public static Text get(Dictionary dictionary, String key, Locale locale, String def) {
```

Comment:
I'd personally like to keep this separate from the text class directly because the text class is just supposed to represent the things supported directly by Minecraft. Can we move this to something like a `TextDict` class?